include/smack/Prelude.h

```diff
@@ -0,0 +1,46 @@
+//
+// This file is distributed under the MIT License. See LICENSE for details.
+//
+
+#ifndef PRELUDE_H
+#define PRELUDE_H
+
+#include "smack/BoogieAst.h"
+#include "smack/SmackRep.h"
+
+#include <string>
+
+namespace smack {
+
+class Prelude {
+  SmackRep &rep;
+
+  FuncDecl* uninterpretedUnaryOp(unsigned width, std::string name);
```

Comment:
It is not clear why any of these are member functions, since they do not access members of the Prelude class — i.e., `rep`. Shouldn’t they just be regular (non-member) functions?